src/bin/validator.rs

```diff
@@ -114,17 +115,27 @@ impl InMemoryKeyManagerConfig for Keys {
 // TODO: This does not belong here, create a new time for AppContextConfig and move to context module
 impl From<&CliOpts> for AppContext {
     fn from(config: &CliOpts) -> Self {
-        let bundler_jwk = if let Some(key_file_path) = &config.bundler_key {
-            let file = fs::read_to_string(key_file_path).unwrap();
-            file.parse().unwrap()
-        } else {
-            let n = config.bundler_public.as_ref().unwrap();
-            public_only_jwk_from_rsa_n(n).expect("Failed to decode bundler key")
+        let bundler_jwk = match (&config.bundler_key, config.bundler_public.as_ref()) {
```

Comment:
This guy will need some merging with https://github.com/Bundlr-Network/validator-rust/pull/42, let me do this